spec/index.html

```diff
@@ -495,6 +496,25 @@ <h2>RDF Documents and Syntaxes</h2>
     and still the subject of group discussion. An alternative to conformance
     levels, "profiles", may be adopted instead, abandoned, or described in 
     another specification.</p>
+
+  <section id="rdf-strings">
+    <h2>Strings in RDF</h2>
+
+    <p>RDF uses Unicode [[Unicode]] as the fundamental representation for string values.
+      Within this, and related specifications, the term <dfn id="dfn-rdf-string">string</dfn>
+      is used to describe an ordered sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a>
+      in any <a data-cite="i18n-glossary#dfn-character-set">character set</a>
+      or <a data-cite="i18n-glossary#dfn-character-encoding">character encoding</a>.
```

Comment:
I think it is gratuitous, and we can just limit it to any character encoding.

```suggestion
      in any <a data-cite="i18n-glossary#dfn-character-encoding">character encoding</a>.
```